sdk/servicebus/service-bus/src/connectionContext.ts

```diff
@@ -27,21 +27,13 @@ export interface ConnectionContext extends ConnectionContextBase {
    * @property A dictionary of ClientEntityContext
    * objects for each of the client in the `clients` dictionary
    */
-  clientContexts: { [name: string]: ClientEntityContext }
+  clientContexts: { [name: string]: ClientEntityContext };
 }
 
 /**
  * @internal
  */
 export namespace ConnectionContext {
-  /**
-   * @property {string} userAgent The user agent string for the ServiceBus client.
-   * See guideline at https://github.com/Azure/azure-sdk/blob/master/docs/design/Telemetry.mdk
-   */
```

Comment:
The link is invalid, do we know where the guidelines have been moved to?

Reply:
@bterlson ?

Reply:
Pretty sure that link was never valid as it would have linked to the nigh-unreadable modoko source.

This is probably the link you want: https://azure.github.io/azure-sdk/general_azurecore.html#telemetry-policy

Reply:
It was there before we published the guidelines to azure.github.io but was later deleted.